docs/getting-started/read-access-control.mdx

```diff
@@ -8,19 +8,113 @@ description: Enforce restrictions on reading data from a network.
 
 In this tutorial, we will cover how to authenticate a user to enforce read access control. This tutorial expands on the schema created in the [Write Access Control](/docs/getting-started/write-access-control) tutorial.
 
-In Kwil, data is primarily read with `view` procedures. For executing `view` procedures, the same general access control rules are available as for [write procedures](/docs/getting-started/write-access-control). However, because data stored on nodes are default public, Kwil networks running in permissioned or federated environments can use the Kwil Gateway to enforce user authentication and stronger read access control.
+In Kwil, data is primarily read with `view` procedures. For executing `view` procedures, the same access control rules are available as for [write procedures](/docs/getting-started/write-access-control).
+
+Additionally, Kwil supports two authentication methods for enforcing read access control:
+- [Private RPC Mode](#private-rpc-mode)
+- [Kwil Gateway](#kwil-gateway)
+
+## Private RPC Mode
+
+Private RPC is a server-side configuration in kwild that enforces user authentication for each call request. Learn more about private mode [here](/docs/node/private-rpc).
+
+Private RPC mode requires a user to sign a message with their [account](/docs/concepts#accounts) keys for each call request. If the signature is valid, the call will return the query result. If the signature is invalid or not provided, the call will return an error.
```

Comment:
We should make it clear that Kwil data is public by default, and so in order to identify someone by their wallet address, they need to sign a message so that we can verify who they are. Also, `account keys` are not a term, and they use a single key. Use the term `private key`.